app/views/support/users/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1 class="govuk-heading-l"><%= @provider.provider_name %></h1>
+
+<%= render TabNavigation::View.new(items: [
```

Comment:
I think we may be able to use the pattern from register to extend a layout which will just have this one component rendered. We do it like this in register for the trainee record page:

https://github.com/DFE-Digital/register-trainee-teachers/blob/master/app/views/layouts/trainee_record.html.erb

I can take a stab at it in my next PR if you want to leave it for now.

Reply:
Good shout. I've implemented the solution your suggested. Might need more refinements later on, but for now, it should do the job.